Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```
@@ -1235,6 +1235,20 @@ CSSTextSpacingEnabled:
     WebCore:
       default: false
 
+CSSTextUnderlinePositionLeftRight:
+  type: bool
+  status: unstable
```

```suggestion
  status: testable
```

Otherwise this won't be enabled in tests